src/main/app/package.json

```diff
@@ -47,11 +47,11 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "^17.3.10",
-    "@angular-eslint/builder": "17.2.1",
-    "@angular-eslint/eslint-plugin": "17.2.1",
-    "@angular-eslint/eslint-plugin-template": "17.2.1",
-    "@angular-eslint/schematics": "^17.2.1",
-    "@angular-eslint/template-parser": "17.2.1",
+    "@angular-eslint/builder": "^17.5.3",
+    "@angular-eslint/eslint-plugin": "^17.5.3",
+    "@angular-eslint/eslint-plugin-template": "^17.5.3",
+    "@angular-eslint/schematics": "^17.5.3",
+    "@angular-eslint/template-parser": "17.5.3",
```

Comment:
I see different versions of the different dependencies here.

Shouldn't this be a group update to angular 17.5.3 ?

Reply:
Yes but Dependabot is broken at the moment (for package.json updates). So that's why I created this one manually. Renovate suggested this update btw.

Reply:
No doubt more stuff needs to be updated..